config/hmri_defaults.m

```diff
@@ -412,4 +411,24 @@
 % Number of Gaussians per tissue class
 hmri_def.proc.nGauss = [2 2 2 3 4 2]; % originally in SPM [1 1 2 3 4 2]
 
+%==========================================================================
+% Experimental features 
+%==========================================================================
+hmri_def.errormaps  = true;
```

Comment:
What did you do here?

Reply:
It allows you to switch on/off creation of hte error maps.

Reply:
:joy: Nice.